src/activity-functions-function-notation.xml

```diff
@@ -105,7 +106,8 @@
 
                     <p>Without knowing anything about function notation, you can still use a language you already know, English, to describe information about the train.</p>
 
-                    <p>For instance:<ul>
+                    <p>For instance:</p>
+                    <ul>
```

Comment:
My comment was supposed to show the tags I used.  Instead it just shows the results. 
The tag order I used is
<p> blah
<ul><li><p>blah</p><li><ul></p>
Is that the right order?